documentation/interpretation_guide.md

```diff
@@ -73,24 +83,32 @@ If there are restrictions for the circulation of private cars based on certain c
 
 ### C8-  International Travel Controls
 
+- This indicator does not have a binary flag variable to reflect geographic variation in policies. This implies that this policy applies generally across the whole jurisdiction. Therefore, we generally code the level of policy that applies everywhere across the jurisdiction- the highest common value of policy nationwide or statewide. We do not code any policies of a higher value that are only available in limited areas.  *For example* if one airport is screening passenger arrivals, but the majority of them are not, this would not be coded as a 1. All airports across the country would need to be screening before this policy could be seen to apply generally and be coded.
```

Comment:
Let's drop the second sentence "This implies...", I don't think it is necessary.